spdockit-spo/spo-snapshots.md

```diff
@@ -38,5 +40,22 @@ Here is what you need to do:
 
 1. Click Next and the loading will start. Wait for the SPDocKit wizard to finish, then SharePoint [farm settings](#internal/get-to-know-spdockit/farm-explorer-screen/farm-explorer-reports) and [permission reports](#internal/get-to-know-spdockit/permissions-reports-screen) will be ready for use!  
 
-    Also, make sure to check our [permission management features](#internal/permission-management/manage-permissions-ribbon-actions) located in __Permissions Explorer__ tab.  
-    Please note that Permission management buttons are enabled only in __live mode__ of Permissions Explorer. There is no need for a snapshot in this case. 
+
+## Automatic Snapshot
+
+SPDocKit Service allows you to configure a schedule to create automatic SharePoint Online snapshots. Once configured, the SPDocKit Service can save your farm settings and send report subscriptions and alerts to selected users, document libraries, or network locations. Click [here](internal/how-to/subscriptions-and-alerts/create-new-subscription) for more information on how to do this.  
+
+It is not required to have Local Admin privilages for a service account. It can be any account, even Local System. See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684190(v=vs.85).aspx) for more information on the Local System account. While setting up the service account via configuration wizard, simply choose what is more convenient to you. 
+
+__Please note:__ Local System account can be used only for SPDocKit service (not possible for SPDocKit Insights Service) if one of the folowing conditions are met:
```

Comment:
_the_ SPDocKit Service?

Reply:
_The Local System account can be used to run the SPDocKit Service (but not the [SPDocKit] Insights Service) if one of the following conditions are met:_

My suggestion. In any case, check the spelling on _following_.